java/registry/src/main/java/io/opensaber/registry/util/OSSystemFieldsHelper.java

```diff
@@ -60,17 +60,17 @@ public void ensureUpdateAuditFields(String title, JsonNode node, String userId)
     public void addSystemProperty(String field, JsonNode node, String userId, String timeStamp) {
         try {
             switch (OSSystemFields.getByValue(field)) {
-                case _osCreatedAt:
-                    OSSystemFields._osCreatedAt.createdAt(node, timeStamp);
+                case osCreatedAt:
```

Comment:
How come there are two files named OSSystemFieldsHelper.java?

Reply:
The another is test class with name OSSystemFieldsHelperTest.java